packages/agents/sdk-wrapper/src/sdkPool.ts

```diff
@@ -41,8 +41,10 @@ export class SdkPool extends SdkShared {
   ): Promise<BigNumber> {
     const _options = options ?? {
       chains: this.config.chains,
+      originProviderUrl: this.config.chains[domainId]?.providers?.[0],
```

Comment:
We need to use fallback provider here, can't rely on first provider implementation.
Ideally we should be using `chains` options for wrapper-sdk and fetch the provider from it under the core-sdk. single_url options for origin and destination were added for any direct request to sdk-server only.

Reply:
Now getting provider from `chains` config that is supplied by client during SDK init.